Userland/Services/RequestServer/ConnectionFromClient.cpp

```diff
@@ -62,6 +194,14 @@ Messages::RequestServer::ConnectNewClientResponse ConnectionFromClient::connect_
     return IPC::File::adopt_fd(socket_fds[1]);
 }
 
+void ConnectionFromClient::enqueue(Work work)
+{
+    Core::deferred_invoke_if([this, work = move(work)] {
```

Comment:
so wait, here we just shuffle the jobs > the 256th around in vectors in the event loop until they can be pushed into the work queue? How is that better than allowing the work queue to expand, or storing them on a secondary queue in this object?

Reply:
e.g. a check to say 

enqueue: if the extra queue has items or we fail to enqueue to the work queue, push to that secondary queue w/it locked. 
worker_main: once queue is empty, try to move items from secondary queue to work queue w/secondary queue locked.

Reply:
we don't have a threadsafe expanding queue, and I don't want to invent one.
I'd rather let the queue grow as well, but no infra

Reply:
re locked secondary queue, no point in using SSPCQ then, might as well just have a single locked queue.